[debug] [2023-01-13T13:09:23.082Z] ----------------------------------------------------------------------
[debug] [2023-01-13T13:09:23.083Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\turtw\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js hosting:disable
[debug] [2023-01-13T13:09:23.084Z] CLI Version:   11.19.0
[debug] [2023-01-13T13:09:23.084Z] Platform:      win32
[debug] [2023-01-13T13:09:23.084Z] Node Version:  v18.5.0
[debug] [2023-01-13T13:09:23.085Z] Time:          Fri Jan 13 2023 21:09:23 GMT+0800 (Singapore Standard Time)
[debug] [2023-01-13T13:09:23.085Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-01-13T13:09:23.094Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects/undefined [none]
[debug] [2023-01-13T13:09:23.540Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects/undefined 403
[debug] [2023-01-13T13:09:23.540Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects/undefined {"error":{"code":403,"message":"The caller does not have permission","status":"PERMISSION_DENIED"}}
[debug] [2023-01-13T13:09:23.542Z] HTTP Error: 403, The caller does not have permission
[debug] [2023-01-13T13:09:23.581Z] FirebaseError: HTTP Error: 403, The caller does not have permission
    at responseToError (C:\Users\turtw\AppData\Roaming\npm\node_modules\firebase-tools\lib\responseToError.js:49:12)
    at RetryOperation._fn (C:\Users\turtw\AppData\Roaming\npm\node_modules\firebase-tools\lib\apiv2.js:288:77)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[error] 
[error] Error: Failed to get Firebase project undefined. Please make sure the project exists and your account has permission to access it.
